vendor/github.com/automationbroker/bundle-lib/clients/kubernetes.go

```diff
@@ -209,8 +210,12 @@ func newKubernetes() (*KubernetesClient, error) {
 	// library
 	clientConfig, err := rest.InClusterConfig()
 	if err != nil {
+		config := os.Getenv("KUBECONFIG")
```

Comment:
This is a different change than we were expecting right? 

can we make sure that this not going to mess something up?

Reply:
That was the 0.2.18 change that never got pulled in. Looks like that was for apb tool support. I'll verify to make sure it doesn't affect the broker.

Reply:
I verified that the above change is used ONLY when the broker is run *outside* of a cluster or if there is an error getting the `InClusterConfig`. I would argue if you get an error getting the `InClusterConfig` and running in a cluster, you have larger problems.

This code is also used when the `apb` tool uses `bundle-lib`.